src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {

```

Comment:
```suggestion
public class Lottery{

```

class Lottery should not extend ColorSupplier as these classes have no common state